src/components/Article.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getArticle } from "../../api";
+import { useParams } from "react-router-dom";
+
+function Article() {
+  const [article, setArticle] = useState([]);
+
+  let { article_id } = useParams();
+
+  useEffect(() => {
+    getArticle(article_id)
+      .then((data) => {
+        console.log(data);
+        setArticle(data.article);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, [article_id]);
+
+  return (
+    <>
+      <section>
+        <h1>{article.title}</h1>
+        <p>By {article.author}</p>
+        <p>Topic: {article.topic}</p>
+        <p>
+          Comments: {article.comment_count} Votes: {article.votes}
+        </p>
+        <img src={article.article_img_url} alt={article.title} />
+
+        <p>{article.body}</p>
+      </section>
+    </>
+  );
+}
```

Comment:
This is a good start, but what is shown to the user while the api is fetching this data? Does the user know that data is being fetched?